server/accounts.go

```diff
@@ -1123,7 +1123,7 @@ func authAccounts(tokenReq bool) []*Account {
 	return nil
 }
 
-// SetAccountResolver will assign the account resolver.
+// SetAccolver will assign the account resolver.
```

Comment:
Revert.

Reply:
The first one  is incorrect but I will revert. Capitalization is wrong

the second is also wierd, seems like a lint/find-replace or something because that one will warn on my ide as well since it doesn' tmatch the function.